app/src/components/custom/tables/RequestsTable.tsx

```diff
@@ -391,33 +405,28 @@ const RequestsTable = (props: RequestTableProps) => {
    * @description Changes the symbol in the CurrencyComparer component when clicked. Also updates the dataManipulator.
    */
   const changeSymbol = () => {
-    const symbolDiv = document.getElementById('symbol');
-    if (symbolDiv) {
-      if (dataManipulator.cost.filter.symbol === Symbols.GT) {
-        setDataManipulator({
-          ...dataManipulator,
-          cost: {
-            ...dataManipulator.cost,
-            filter: {
-              ...dataManipulator.cost.filter,
-              symbol: Symbols.LT,
-            },
+    if (dataManipulator.cost.filter.symbol === Symbols.GT) {
+      setDataManipulator({
+        ...dataManipulator,
+        cost: {
+          ...dataManipulator.cost,
+          filter: {
+            ...dataManipulator.cost.filter,
+            symbol: Symbols.LT,
```

Comment:
Looks like instead of repeating the setDataManipulator function call here you could have a `const changedCostFilterSymbol = dataManipulator.cost.filter.symbol === Symbols.GT ? Symbols.LT : Symbols.GT;` and just set `symbol` property to this variable when you call the function.

Reply:
Yes, it does seem that way. Changing now.